### PR TITLE
New interface for Travis.genkeys

### LINKED ISSUE #6: Travis.genkeys(): don't take arguments / drop Git
`Travis.genkeys()` used to write a `.documenter` key file to the repository, so it needed to to which package the key was for. But it has not actually touched anything in the repo for a long time, so it does not actually need to take an argument.

Relatedly, Git is used to figure out the exact remote URLs from of the repository. But those are simply printed out for the user, and can probably replaced with generic ones like `https://github.com/USER/REPOSITORY/settings/keys`. So it might be better to just drop Git usage entirely before #5.

Alternatively, we could still have a `Travis.genkeys(::Module)` in addition to `Travis.genkeys()`, which still tries to figure out the URLs, to help the user as much as possible.

### PR DESCRIPTION
**Pull request by mortenpi**
Brings `Travis.genkeys` into the Pkg3 era. Closes #6.

### CODE REVIEW

File: src/Travis.jl [hunk 1]
```diff
@@ -6,93 +6,143 @@ $(EXPORTS)
...
+This method requires the following command lines programs to be installed:
+
+- `which`
+- `git`
+- `ssh-keygen`
+
+!!! note
+    The package must be in development mode. Make sure you run
+    `pkg> develop pkg` from the Pkg REPL, or `Pkg.develop(\"pkg\")`
+    before generating docs.
```
Comment (fredrikekre) [inline src/Travis.jl]: `docs` -> `ssh keys`

File: src/Travis.jl [hunk 2]
```diff
@@ -6,93 +6,143 @@ $(EXPORTS)
...
+
+# Examples
+
```
Comment (fredrikekre) [inline src/Travis.jl]: remove empty line?

File: src/Travis.jl [hunk 3]
```diff
@@ -6,93 +6,143 @@ $(EXPORTS)
...
 - `which`
-- `git`
-- `travis`
 - `ssh-keygen`
 
 # Examples
 
-```jlcon
-julia> using Documenter
+```julia-repl
```
Comment (fredrikekre) [inline src/Travis.jl]: Remove empty line between `# Examples ` and code block?

File: src/Travis.jl [hunk 4]
```diff
@@ -6,93 +6,143 @@ $(EXPORTS)
...
+julia> using DocumenterTools
+
+julia> Travis.genkeys()
+[ Info: add the public key below to https://github.com/\$USER/\$REPO/settings/keys with read/write access:
+
+ssh-rsa AAAAB3NzaC2yc2EAAAaDAQABAAABAQDrNsUZYBWJtXYUk21wxZbX3KxcH8EqzR3ZdTna0Wgk...jNmUiGEMKrr0aqQMZEL2BG7 username@hostname
```
[the remark this answers is not in the document]
Comment (mortenpi) [reply src/Travis.jl]: Nah, I was too lazy to generate them, so I just took the ones for Base. What could possibly go wrong..? :laughing: 

Seriously though: yes, they are freshly generated. Also, a big chunk of the key is actually missing (see the ellipsis). Having a full key there makes the example code block unhelpfully large I found.

File: src/Travis.jl (hunk 2 above)
Comment (mortenpi) [reply src/Travis.jl]: I usually keep the headings separated with empty lines. I find it helps with readability.

**Pull request merged**